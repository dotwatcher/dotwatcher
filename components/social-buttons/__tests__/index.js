import 'jest-styled-components';

import React from 'react';
import SocialButtons from '../index';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

describe('<SocialButtons />', () => {
	it('renders <SocialButtons />', () => {
		const tree = renderer.create(<SocialButtons />).toJSON()
		expect(tree).toMatchSnapshot()
	});
});