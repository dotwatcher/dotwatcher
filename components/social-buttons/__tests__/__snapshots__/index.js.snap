// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SocialButtons /> renders <SocialButtons /> 1`] = `
.c0 {
  background-color: #3b5998;
  border: 0;
  color: #fff;
}

.c1 {
  background-color: #00aced;
  border: 0;
  color: #fff;
}

<div>
  <button
    className="c0"
    onClick={[Function]}
    target="_blank"
  >
    Share
  </button>
  <button
    className="c1"
    onClick={[Function]}
    target="_blank"
  >
    Tweet
  </button>
</div>
`;
